dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -1130,6 +1132,8 @@ public void testFindLinkageProblems_grpcAndGuava() throws IOException {
   public void testFindLinkageProblems_referenceToJava11Method() throws IOException {
     // protobuf-java 3.12.4 references a Java 11 method that does not exist in Java 8
     // https://github.com/protocolbuffers/protobuf/issues/7827
+    Assume.assumeThat(System.getProperty("java.version"), startsWith("1.8.0"));
```

Comment:
Static import reads well, compared with `StringStartsWith.startsWith`.

Reply:
why not use java.lang.String.startsWith()? In not, because that method is much more familiar, this really can't be a static import.

Reply:
`String.startsWith` would give a message `got false expected true`.

This `StringStartsWith.startsWith` gives more clear message: `got: "11.0.4", expected: a string starting with "1.8.0"`.

Reply:
Fair enough. Static import is still not a good idea here given the confusion with the better known String.startsWith method.